sunpy/net/scraper.py

```diff
@@ -14,23 +14,26 @@
 from bs4 import BeautifulSoup
 from dateutil.relativedelta import relativedelta
 
-from astropy.time import Time
-
 from sunpy import log
 from sunpy.extern.parse import parse
 from sunpy.time import TimeRange
-from sunpy.util.exceptions import warn_user
 
 __all__ = ['Scraper']
 
-# regular expressions to convert datetime format
+# parse expressions to convert into datetime format
 # added `%e` as for milliseconds `%f/1000`
```

Comment:
Is this comment true? If not, we should remove it.

Reply:
yeah, it's true, we use that list for pattern -> timepattern conversions like parse patterns like "{year:4d}" to %Y

Reply:
Should it be here and not be above line 32?

Reply:
oh yes, the comment about adding milliseconds can be added right by where we mention it.